Packs/FeedDShield/Integrations/FeedDShield/FeedDShield.py

```diff
@@ -0,0 +1,36 @@
+from CommonServerPython import *
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+
+    params['indicator_type'] = FeedIndicatorType.CIDR
+
+    params['url'] = 'https://www.dshield.org/block.txt'
+    params['ignore_regex'] = "[#S].*"
+    params['indicator'] = json.dumps({
+        "regex": r"^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\t[\d.]*\t(\d{1,2})",
+        "transform": "\\1/\\2"
+    })
+
+    fields = json.dumps({
+        "geocountry": {
+            "regex": "^.*\\t.*\\t[0-9]+\\t[0-9]+\\t[^\\t]+\\t([A-Z]+)",
+            "transform": "\\1"
+        },
+        "registrarabuseemail": {
+            "regex": "^.*\\t.*\\t[0-9]+\\t[0-9]+\\t[^\\t]+\\t[A-Z]+\\t(\\S+)",
+            "transform": "\\1"
+        }
+    })
+    params['fields'] = fields
+
```

Comment:
saw that you've deleted some fields from here, why?

Reply:
talking to @David-BMS - we decided these fields are not relevant. So I took them off.

Reply:
it sounds like minemeld parsed them so we should too, lets add them back.
Users will decide what to do with them on the mapping front

Reply:
done